templates/vs/csharp/custom-copilot-travel-agent/m365agents.yml.tpl

```diff
@@ -0,0 +1,135 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.11/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.11
+
+additionalMetadata:
+  sampleTag: TeamsFx-Samples:travel-agent
```

Comment:
Remove the sample tag as this is now a template instead of sample

Reply:
removed